content/src/controller/Controller.ts

```diff
@@ -412,7 +412,7 @@ export class Controller {
     // Path: /deployments
     // Query String: ?from={timestamp}&toLocalTimestamp={timestamp}&entityType={entityType}&entityId={entityId}&onlyCurrentlyPointed={boolean}&deployedBy={ethAddress}
 
-    const stringEntityTypes = this.asArray<string>(req.query.entityType as string)
+    const stringEntityTypes = this.asArray<string>(req.query.entityType as string | string[])
```

Comment:
can this turn into an array of arrays?

Reply:
As I can see in the method `asArray`, it shouldn't:

```ts
export function asArray<T>(elements: T[] | T): T[] {
  if (!elements) {
    return []
  }
  if (elements instanceof Array) {
    return elements
  }
  return [elements]
}
```